src/app_service/service/network/dto.nim

```diff
@@ -9,7 +9,7 @@ type NetworkDto* = ref object
   layer* {.serializedFieldName("layer").}: int
   chainName* {.serializedFieldName("chainName").}: string
   rpcURL* {.serializedFieldName("rpcUrl").}: string
-  fallbackURL* {.serializedFieldName("fallbackUrl").}: string
+  fallbackURL* {.serializedFieldName("fallbackURL").}: string
```

Comment:
@Khushboo-dev-cpp using `serializedFieldName` is not necessary in whole this file.

Reply:
This is needed as we convert incoming json to NetworkDto type!